File: solutions/go/scrabble-score/1/scrabble_score.go
package scrabble

import "strings"

func Score(word string) int {
	var count = 0
	word = strings.ToLower(word)
	for _, rune := range word {
		switch {
		case strings.ContainsRune("aeioulnrst", rune):
			count += 1
		case strings.ContainsRune("dg", rune):
			count += 2
		case strings.ContainsRune("bcmp", rune):
			count += 3
		case strings.ContainsRune("fhvwy", rune):
			count += 4
		case strings.ContainsRune("k", rune):
			count += 5
		case strings.ContainsRune("jx", rune):
			count += 8
		case strings.ContainsRune("qz", rune):
			count += 10
		}
	}
	return count
}

File: solutions/python/scrabble-score/1/scrabble_score.py
def score(word):
    scores = {
        "AEIOULNRST": 1,
        "DG": 2,
        "BCMP": 3,
        "FHVWY": 4,
        "K": 5,
        "JX": 8,
        "QZ": 10,
    }
    for key, value in tuple(scores.items()):
        del scores[key]
        scores.update({character: value for character in key})
    return sum([scores[character] for character in word.upper()])
